tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1623,4 +1623,55 @@ class CompletionSuite extends BaseCompletionSuite {
     topLines = Some(1),
   )
 
+  check(
+    "type-with-params",
+    s"""|object O {
+        | type TTT[A <: Int] = List[A]
```

Comment:
I wonder if this would be an issue with match types, whether we would not show too much for the user if we show the right hand side

Reply:
![Screenshot 2023-08-02 at 13 42 11](https://github.com/scalameta/metals/assets/26606662/c85c2f43-1238-4076-9256-225f4f47f727)
Or do we want just the result type in the description?

Reply:
Hmm... since it's in the detail maybe that's a boon? Let's leave it as is

Reply:
The question was if you want the detail to be
```
[type params] = rhs
```
as currently, or
```
rhs
```
or possibly
```
= rsh
```

Reply:
Maybe `= rhs`?